core/server/models/base/index.js

```diff
@@ -717,19 +712,19 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
         }
 
         return itemCollection.fetchPage(options).then(function formatResponse(response) {
-            var data = {},
-                models;
+            const data = {};
+            data.meta = {pagination: response.pagination};
 
-            options.columns = requestedColumns;
-            models = response.collection.toJSON(options);
+            // Filtering attributes here so they are not leaked to api layer
+            // where models are serialized to json and do not do more filtering
+            data[tableName] = response.collection.models.map((model) => {
```

Comment:
🤔 I think `findPage` should not return `{posts: x}` - this is an api format.

Reply:
Just thought about this again. Def should happen in controller level.
e.g. imagine we want to return pages and posts. The API level defines the format.

Reply:
How would we return the meta in this case?

Reply:
There are different ways to solve this

e.g. `{data: data, meta: meta}`

Would be helpful if you could come up with some more ideas :) And share them here.